packages/core/platform/bundles/paas-full.yaml

```
@@ -155,6 +155,12 @@ releases:
   privileged: true
   dependsOn: [piraeus-operator,cilium,kubeovn,cert-manager]
 
+- name: snapshot-controller
+  releaseName: snapshot-controller
+  chart: cozy-snapshot-controller
+  namespace: cozy-snapshot-controller
+  dependsOn: [piraeus-operator,cilium,kubeovn,cert-manager,linstor]  
```

does it really depends on `piraeus-operator` and `linstor`? I think it can be managed separately

cert-manager should be replaced to `cert-manager-issuers`
```suggestion
  dependsOn: [cilium,kubeovn,cert-manager-issuers]  
```